pupil-spa/src/app/familiarisation-area/familiarisation-area.component.scss

```diff
@@ -0,0 +1,114 @@
+.font-xsmall {
+    font-size: 12px

```

Comment:
missing ';'

Reply:
Nice catch - it would be good to add a style linter. We would catch a lot of problems like these.